dreem/models/transformer.py

```
@@ -79,28 +81,69 @@ def __init__(
         self.temp_emb = Embedding(emb_type="off", mode="off", features=self.d_model)
 
         if self.embedding_meta:
+            self.embedding_agg_method = (
+                embedding_meta["embedding_agg_method"]
+                if "embedding_agg_method" in embedding_meta
+                else "average"
```

Ruff

Use `embedding_meta.get("embedding_agg_method", "average")` instead of an `if` block

Replace with `embedding_meta.get("embedding_agg_method", "average")`

(SIM401)